packages/node-resolve/src/index.js

```diff
@@ -25,8 +27,9 @@ const defaults = {
   extensions: ['.mjs', '.js', '.json', '.node'],
   resolveOnly: []
 };
+export const DEFAULTS = deepFreeze(defaults);
```

Comment:
you missed something here. go back and read my comment again :)

Reply:
Spreading the object will do what you want, right? 🤔

```suggestion
export const DEFAULTS = deepFreeze({ ...defaults });
```

Reply:
No.

```js
→ node
Welcome to Node.js v12.15.0.
Type ".help" for more information.
> const a = { b: [0] }
undefined
> const b = { ...a }
undefined
> b
{ b: [ 0 ] }
> b.b.push(1)
2
> b.b
[ 0, 1 ]
> a.b
[ 0, 1 ]
```

You're not catching onto the differences between a shallow copy and a deep copy. In a shallow copy, any properties which contain objects are by-reference, meaning a change to them also affects the "parent." 

Please try these out in your local REPL. The REPL comes in handy to demo ideas before proposing them